pipenv/core.py

```diff
@@ -2303,6 +2303,8 @@ def do_run(command, args, three=None, python=False, pypi_mirror=None):
     inline_activate_virtual_environment()
     try:
         script = project.build_script(command, args)
+        cmd_string = ' '.join([script.command] + script.args)
+        logging.getLogger("pip").info("Run: '{0}'".format(cmd_string))
```

Comment:
This logger is not used in any other place of this file. To keep consistency, I recommend
```python
if environments.is_verbose():
    click.echo("Run: {!0r}".format(cmd_string), err=True)
```

Reply:
Sure.

Reply:
(as a sidenote, logging using a logger implementation isn't a great practice, especially in our case -- we parse the output that goes to stdout in many cases)